### PR TITLE
Added zoom for PanelEditor

### PR DESCRIPTION
**Pull request by SamirMokiem**
Implements #184

- [x]  Update french translation

- [ ]  Bump version

### CODE REVIEW

File: src/main/resources/lang/fr_fr.json [hunk 1]
```diff
@@ -55,6 +55,9 @@
 	"popup_menu.forward": "Restaurer",
 	"popup_menu.mark_deobfuscated": "Marquer comme déobfusqué",
 	"popup_menu.reset_obfuscated": "Réinitialiser à obfusqué",
+	"popup_menu.zoom.in": "Zoom in",
+	"popup_menu.zoom.out": "Zoom out",
+	"popup_menu.zoom.reset": "Reset zoom",
```
Comment (Runemoro) [inline src/main/resources/lang/fr_fr.json:60]: `Agrandir`, `Rapetisser`, `Réinitialiser la grandeur`

Comment (Runemoro) [reply src/main/resources/lang/fr_fr.json:60]: Or maybe @Yanis48 do you have some better suggestions?

File: src/main/java/cuchaz/enigma/gui/panels/PanelEditor.java [hunk 2]
```diff
@@ -17,11 +17,13 @@
 
 public class PanelEditor extends JEditorPane {
 	private boolean mouseIsPressed = false;
+	public Integer fontSize = 12;
```
Comment (Juuxel) [inline src/main/java/cuchaz/enigma/gui/panels/PanelEditor.java:20]:
```suggestion
	public int fontSize = 12;
```

File: src/main/java/cuchaz/enigma/gui/panels/PanelEditor.java [hunk 3]
```diff
@@ -135,6 +146,19 @@ public void keyReleased(KeyEvent event) {
 		});
 	}
 
+	public void setEditorZoom(Integer zoomAmount) {
```
Comment (Juuxel) [inline src/main/java/cuchaz/enigma/gui/panels/PanelEditor.java:149]: It doesn't really set the zoom level but offsets it. Also, this should be using `int` as well.

File: src/main/resources/lang/fr_fr.json (hunk 1 above)
Comment (Yanis48) [reply src/main/resources/lang/fr_fr.json:60]: Zoomer, Dézoomer, Réinitialiser le zoom

**Pull request merged**